tests/python_package_test/test_dask.py

```diff
@@ -1224,6 +1224,51 @@ def test_training_succeeds_when_data_is_dataframe_and_label_is_column_array(
     client.close(timeout=CLIENT_CLOSE_TIMEOUT)
 
 
+@pytest.mark.parametrize('task', tasks)
+@pytest.mark.parametrize('output', data_output)
+@pytest.mark.parametrize('init_score', [0.25, 0.75])
+def test_init_score(
+        task,
+        output,
+        init_score,
+        client):
+    if task == 'ranking' and output == 'scipy_csr_matrix':
+        pytest.skip('LGBMRanker is not currently tested on sparse matrices')
+
+    if task == 'ranking':
+        _, _, _, _, dX, dy, dw, dg = _create_ranking_data(
+            output=output,
+            group=None
+        )
+        model_factory = lgb.DaskLGBMRanker
+    else:
+        _, _, _, dX, dy, dw = _create_data(
+            objective=task,
+            output=output,
+        )
+        dg = None
+        if task == 'classification':
+            model_factory = lgb.DaskLGBMClassifier
+        elif task == 'regression':
+            model_factory = lgb.DaskLGBMRegressor
+
+    params = {
+        'n_estimators': 1,
+        'num_leaves': 2,
+        'time_out': 5
+    }
+    if output.startswith('dataframe'):
+        init_scores = dy.map_partitions(lambda x: pd.Series([init_score] * x.size))
+    else:
+        init_scores = da.full_like(dy, fill_value=init_score, dtype='float')
```

Comment:
oh cool! did not know about https://docs.dask.org/en/latest/array-api.html#dask.array.full_like

Reply:
Yeah I wanted something like that for the dataframe too haha. Maybe the `if` statement would be more consistent if I used `map_blocks(lambda x: np.asarray([init_score] * x.size))`. What do you think?

Reply:
yeah fair point! I have no strong preference either way, so whichever you prefer is good with me.